chains/solana/contracts/tests/ccip/ccip_router_test.go

```diff
@@ -2785,7 +2785,7 @@ func TestCCIPRouter(t *testing.T) {
 			}
 		})
 
-		t.Run("hen sending a Valid CCIP Message but the user does not have enough funds of the fee token, it fails", func(t *testing.T) {
+		t.Run("When sending a Valid CCIP Message but the user does not have enough funds of the fee token, it fails", func(t *testing.T) {
```

Comment:
Should we add a test for this specific case?

Reply:
I thought about it initially, but that requires me to use another program as the caller (so that it's a PDA) and guarantee that the balance is just right. It seems like an overkill to create a new program (or extend the `ccip-receiver`) just to test this fairly basic check.

Reply:
Yes, let's avoid having more ccip receiver examples